src/main/scala/io/delta/tables/DeltaTable.scala

```diff
@@ -522,4 +521,65 @@ object DeltaTable {
     }
   }
 
+  /**
+   * :: Evolving ::
+   *
+   * Check if the provided path is the root of a Delta table using the given SparkSession.
+   *
+   */
+  @Evolving
+  def isDeltaTable(sparkSession: SparkSession, path: Path): Boolean = {
+    DeltaTableUtils.isDeltaTable(sparkSession, path)
+  }
+
+  /**
+   * :: Evolving ::
+   *
+   * Check if the provided path is the root of a Delta table.
+   *
+   * Note: This uses the active SparkSession in the current thread to search for the table. Hence,
+   * this throws error if active SparkSession has not been set, that is,
+   * `SparkSession.getActiveSession()` is empty.
+   *
+   */
+  @Evolving
+  def isDeltaTable(path: Path): Boolean = {
+    val sparkSession = SparkSession.getActiveSession.getOrElse {
+      throw new IllegalArgumentException("Could not find active SparkSession")
+    }
+
```

Comment:
nit: this line can be removed

Reply:
I am not sure if we need a separate `isDeltaTable` for parameter type `Path`, we can perhaps stick to String to be uniform with `forPath` method. In that case you would need to change the method above this to take `String` as well.

Reply:
That makes sense to me. I've addressed all comments you left :)